src/kibana/plugins/settings/sections/objects/_objects.js

```diff
@@ -12,42 +16,52 @@ define(function (require) {
   .directive('kbnSettingsObjects', function (config, Notifier, Private, kbnUrl) {
     return {
       restrict: 'E',
-      controller: function ($scope, $injector, $q, AppState) {
+      controller: function ($scope, $injector, $q, AppState, es) {
+        var notify = new Notifier({ location: 'Saved Objects' });
 
         var $state = $scope.state = new AppState();
-
-        var resetCheckBoxes = function () {
-          $scope.deleteAll = false;
-          _.each($scope.services, function (service) {
-            _.each(service.data, function (item) {
-              item.checked = false;
-            });
-          });
-        };
+        $scope.currentTab = null;
+        $scope.selectedItems = [];
 
         var getData = function (filter) {
```

Comment:
Since you have familiarized yourself with this code, would you mind giving some more context to what "data" is here? From what I can tell everywhere but here "data" is referring to "hits", but here "data" is referring to a section type?

Sidenote: Since the method is returning an object, perhaps we should give that object a definition (either in comments or a class).